odoo/addons/base/models/ir_ui_view.py

```diff
@@ -576,7 +576,7 @@ def default_view(self, model, view_type):
         :rtype: int
         """
         domain = [('model', '=', model), ('type', '=', view_type), ('mode', '=', 'primary')]
-        return self.search(domain, limit=1).id
+        return self.search(domain, limit=1, order='priority,id').id
```

Comment:
Not that I think about it, I'm not sure that we shouldn't change the default order instead? :thinking: 
Usually we reorder using the handle widget, and that is the only way to know the order is by looking at it. Maybe not relevant for views though...

Reply:
I will do the minimum fix in this pr.
I will see what it implies if I change the order of ir.ui.view in another pr